books/migrations/0003_change_author_table_&apply_data_migration.py

```diff
@@ -0,0 +1,63 @@
+# Generated by Django 3.2.5 on 2021-07-13 20:21
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def populate_long_names(apps, schema_editor):
```

Comment:
rename this to `populate_short_names`

Reply:
oh my bad :p was doing some own experimentation and forgot to change the name :P

Reply:
only this is remaining change, rest looks good :)